src/lib.rs

```diff
@@ -74,11 +74,14 @@
 pub mod options;
 
 mod bson_util;
+#[cfg(any(feature = "tokio-runtime", feature = "async-std-runtime"))]
```

Comment:
I added annotations to each module that will need to use `RUNTIME` either directly or transitively so that users who don't have tokio xor async-std enabled will only get the custom compiler error message we give below.

Reply:
Actually, I missed a bunch; literally the only module we have that isn't in this set is `collation`.